src/main/scripts/org/reaktivity/specification/http2/rfc7540/connection.management/http.post.exchange.with.large.payload.before.settings/client.rpt

```diff
@@ -0,0 +1,103 @@
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property data ${http:randomBytes(16384)}
+property data1 ${http:randomBytes(10000)}
+
+connect await ROUTED_SERVER
+        "nukleus://streams/http2#0"
+        option nukleus:window 32768
+        option nukleus:transmission "duplex"
+connected
+
+# client connection preface
+write "PRI * HTTP/2.0\r\n"
+      "\r\n"
+      "SM\r\n"
+      "\r\n"
+write flush
+
+write [0x00 0x00 0x0c]                   # length = 12
+      [0x04]                             # HTTP2 SETTINGS frame
+      [0x00]                             # flags = 0x00
+      [0x00 0x00 0x00 0x00]              # stream_id = 0
+      [0x00 0x03 0x00 0x00 0x00 0x64]    # SETTINGS_MAX_CONCURRENT_STREAMS(0x03) = 100
+      [0x00 0x04 0x00 0x00 0xff 0xff]    # SETTINGS_INITIAL_WINDOW_SIZE(0x04) = 65535
+write flush
+
+write [0x00 0x00 0x13]                  # length = 19
+      [0x01]                            # HEADERS frame
+      [0x04]                            # END_HEADERS
+      [0x00 0x00 0x00 0x01]             # stream_id = 1
+      [0x83]                            # :method: POST
+      [0x86]                            # :scheme: http
+      [0x84]                            # :path: /
+      [0x01] [0x0e] "localhost:8080"    # :authority: localhost:8080
+      [0x00 0x40 0x00]                  # length = 16384
+      [0x00]                            # HTTP2 DATA frame
+      [0x00]                            # NO_END_STREAM
+      [0x00 0x00 0x00 0x01]             # stream_id = 1
+      ${data}
+      [0x00 0x40 0x00]                  # length = 16384
+      [0x00]                            # HTTP2 DATA frame
+      [0x00]                            # NO_END_STREAM
+      [0x00 0x00 0x00 0x01]             # stream_id = 1
+      ${data}
+      [0x00 0x27 0x10]                  # length = 10000
+      [0x00]                            # HTTP2 DATA frame
+      [0x01]                            # END_STREAM
+      [0x00 0x00 0x00 0x01]             # stream_id = 1
+      ${data1}
+write flush
+
+# server connection preface - SETTINGS frame
+read [0x00 0x00 0x12]                   # length = 18
+     [0x04]                             # HTTP2 SETTINGS frame
+     [0x00]                             # flags = 0x00
+     [0x00 0x00 0x00 0x00]              # stream_id = 0
+     [0x00 0x03 0x00 0x00 0x00 0x64]    # SETTINGS_MAX_CONCURRENT_STREAMS(0x03) = 100
+     [0x00 0x04 0x00 0x00 0x00 0x00]    # SETTINGS_INITIAL_WINDOW_SIZE(0x04) = 0
+     [0x00 0x06 0x00 0x00 0x20 0x00]    # SETTINGS_MAX_HEADER_LIST_SIZE(0x06) = 8192
+
+read [0x00 0x00 0x00]                   # length = 0
+     [0x04]                             # HTTP2 SETTINGS frame
+     [0x01]                             # ACK
+     [0x00 0x00 0x00 0x00]              # stream_id = 0
```

Comment:
Please move the `read SETTINGS` and `read SETTINGS ack` before `write HEADERS`.

Only `write SETTINGS ack` needs to be after `write HEADERS` for this test scenario.